airflow/providers/common/sql/operators/sql.py

```diff
@@ -73,6 +73,8 @@ def _get_failed_checks(checks, col=None):
 
 parse_boolean = _parse_boolean
 """
+:sphinx-autoapi-skip:
```

Comment:
Seemed like this is only pertinent for internal/maintainer use and shouldn't _really_ be in external Python API documentation.